packages/service/src/service.ts

```diff
@@ -56,6 +64,23 @@ app.use(passport.session());
 
 app.post("/login", passport.authenticate("local", { successRedirect: "/" }));
 
+app.put("/users/update-password", async function (req, resp) {
+  await updatePassword(req.user);
```

Comment:
It's not clear how params are getting passed through here. How can we  take params from the request body?

See the documentation here: http://expressjs.com/en/resources/middleware/body-parser.html#bodyparserjsonoptions